### PR TITLE
Added time range limit for GetLogs function. Fixed logs time pulling …

### PR DESCRIPTION
**Pull request by kononovn**
…issue

### CODE REVIEW

File: test/util/pod/pod.go [hunk 1]
```diff
@@ -123,8 +123,8 @@ func ExecCommand(cs *testclient.ClientSet, pod *corev1.Pod, command ...string) (
 }
 
 // GetLog connects to a pod and fetches log
-func GetLog(cs *testclient.ClientSet, p *corev1.Pod) (string, error) {
-	req := cs.Pods(p.Namespace).GetLogs(p.Name, &corev1.PodLogOptions{})
+func GetLog(cs *testclient.ClientSet, p *corev1.Pod, s int64) (string, error) {
```
Comment (fedepaol) [inline test/util/pod/pod.go:126]: Can you use time.Duration?

File: test/util/cluster/cluster.go [hunk 2]
```diff
@@ -159,7 +159,7 @@ func CheckReadyGeneration(clients *testclient.ClientSet, operatorNamespace strin
 		return false, nil
 	}
 
-	logs, err := pods.GetLog(clients, podObj)
+	logs, err := pods.GetLog(clients, podObj, 300)
```
Comment (fedepaol) [inline test/util/cluster/cluster.go:162]: 5 * time.Minute

**Comment by SchSeba** [comment]
@kononovn please fix the commit message and the PR tittle

File: test/util/cluster/cluster.go (hunk 2 above)
Comment (kononovn) [reply test/util/cluster/cluster.go]: I tried to fix it. Please take a look

File: test/util/pod/pod.go (hunk 1 above)
Comment (kononovn) [reply test/util/pod/pod.go]: I tried to fix it. Please take a look

File: test/util/pod/pod.go [hunk 3]
```diff
@@ -123,8 +124,9 @@ func ExecCommand(cs *testclient.ClientSet, pod *corev1.Pod, command ...string) (
 }
 
 // GetLog connects to a pod and fetches log
-func GetLog(cs *testclient.ClientSet, p *corev1.Pod) (string, error) {
-	req := cs.Pods(p.Namespace).GetLogs(p.Name, &corev1.PodLogOptions{})
+func GetLog(cs *testclient.ClientSet, p *corev1.Pod, s time.Duration) (string, error) {
+	sinceSeconds := int64(s.Seconds())
```
Comment (fedepaol) [inline test/util/pod/pod.go:128]: You can use `logStart := metav1.NewTime(s)` and use the `SinceTime` PodLogOption: `SinceTime: &logStart`

Comment (fedepaol) [reply test/util/pod/pod.go:128]: Never mind, it's fine this way.

Comment (kononovn) [reply test/util/pod/pod.go:128]: Fixed Var name

**Pull request merged**